### DISCUSSION TITLE
Sick App

### DISCUSSION THREAD

**Discussion by alexander-bergeron** [noah-nuebling/mac-mouse-fix#620] [Other]
Love what this app can do, I don't love how I am forced to adopt all the functionality (such as click drag). I will take a look at the source and try to contrite. Still want to say thanks for such a great app.

**Comment by JulianKingman** [comment]
Came here to say the same thing. This app is incredible, and I love that you (author) created a license to give people freedom with the code.

**Comment by noah-nuebling** [comment]
Thanks guys! I'm glad you're enjoying the app.

@alexander-bergeron You can disable the Click and Drag feature by clicking the little minus (-) button on the left.

Does that solve your problem or did you have something else in mind?

Let me know if you have any more questions feedback.
